apps/explorer/src/app/routes/parties/id/components/lib/combine-deposits-withdrawals.test.ts

```diff
@@ -0,0 +1,69 @@
+import { getDepositWithdrawalStatusLabel } from '../party-deposits-withdrawals-status-icon';
+import { isDepositStatus } from './combine-deposits-withdrawals';
+import { DepositStatus, WithdrawalStatus } from '@vegaprotocol/types';
+
+describe('isDepositStatus & getDepositWithdrawalStatusLabel', () => {
+  it('Use deposit status label for open, regardless of type', () => {
+    expect(isDepositStatus(DepositStatus['STATUS_OPEN'])).toBe(true);
```

Comment:
non blocking: I find these to be overkill given we are using typescript and we inherently test this via getDepositWithdrawalStatusLabel

Reply:
I can see that point of view, but these are testing (and demonstrating via tests) the situation that leads to the label. Even with the type narrowing the shared labels will come out as a `DepositStatus`, and that's expected and handled correctly (even if it seems surprising). I'm going to leave these as is.